AK/NumberFormat.cpp

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
This commit isn't atomic. Make another commit to move the definitions to the cpp file.